src/test/java/org/folio/rest/impl/VoucherNumberTest.java

```diff
@@ -45,12 +47,30 @@ public void testGetVoucherNumber() throws MalformedURLException {
     voucherNumberList.add(getNumberAsLong());
   }
 
+  @Test
+  public void testCurrentStartValueVoucherNumber() throws MalformedURLException {
+    int voucherNumberSequenceInitialValue = 3;
+    while(voucherNumberSequenceInitialValue <= 5) {
+      voucherNumberList.add(getNumberAsLong());
+      voucherNumberSequenceInitialValue++;
```

Comment:
If I understand correctly, sequence start value should be invariant in two cases: 1) between initialization and call of POST /voucher-storage/voucher-number/start/{value}; 2) between two calls of POST /voucher-storage/voucher-number/start/{value}. I suppose positive test should contains both these scenarious.

Reply:
I have added tests to handle both scenarios.